websites/R/Replit/presence.ts

```diff
@@ -4,7 +4,7 @@ const presence = new Presence({
 
 presence.on("UpdateData", () => {
   const presenceData: PresenceData = {
-      largeImageKey: "repl"
+      largeImageKey: "replit"
```

Comment:
If you've changed the `largeImageKey`, you need to change it in the application in the dev portal (which requires you to contact the owner of the presence, which I doubt you have done).